package.json

```diff
@@ -58,13 +68,15 @@
     "browserstack": "grunt build && sh build/run-browserstack.sh",
     "build": "grunt build",
     "test": "grunt",
-    "coverage": "grunt coverage"
+    "coverage": "grunt coverage",
+    "test:cli": "grunt build && npm link && qunit test/cli/*.js"
```

Comment:
@trentmwillis You shouldn't need `npm link`: npm will prepend `./node_modules/.bin` to PATH when this npm script is run.